src/pcapi/notifications/sms/sending_limit.py

```diff
@@ -0,0 +1,21 @@
+from redis import Redis
+
+from pcapi import settings
+from pcapi.models import User
+
+
+def _sent_SMS_count_key_name(user: User):
+    return f"sent_SMS_count_user_{user.id}"
+
+
+def update_sent_SMS_count(redis: Redis, user: User):
+    key_name = _sent_SMS_count_key_name(user)
+    count = redis.incr(key_name)
+    if count == 1:
+        redis.expire(key_name, 30 * 24 * 60 * 60)
+
+
+def is_SMS_sending_allowed(redis: Redis, user: User):
+    sent_SMS_count = redis.get(_sent_SMS_count_key_name(user))
+
+    return not sent_SMS_count or int(sent_SMS_count) < settings.MAX_SMS_SENT_FOR_PHONE_VALIDATION
```

Comment:
Est-il nécessaire de convertir avec `int()` ce que renvoie Redis ? J'ai l'impression qu'on reçoit déjà bien un entier, ici. Ça donnerait :

```suggestion
    return not sent_SMS_count or sent_SMS_count < settings.MAX_SMS_SENT_FOR_PHONE_VALIDATION
```

Reply:
Pas si sûr 

```python
import redis

r = redis.Redis(host='localhost', port=6379, db=0)

r.incr('foo')
r.get('foo') # -> b'1'

```

Ce qui correspond à [la documentation](https://redis.io/commands/get) 

> **Return value**
> Bulk string reply: the value of key, or nil when key does not exist.

Reply:
Oh, bien vu, tu as raison. J'avais testé `incr()` (qui renvoie un `int`), mais je n'avais pas testé `get()` qui renvoie effectivement une string.

Reply:
oui, redis renvoie une string